### ISSUE TITLE
LLama Group, Winamp owner clearly stated 

### ISSUE THREAD

**Issue by Devilsdev1985** [WinampDesktop/winamp#225]
That Winamp will stay free (see press release hereafter). They opened the source code to allow the community to further develop the historical player because they won't.

Yet, they need to keep control on what it becomes because Winamp has other projects ongoing and they must care about the security of users and quality of Winamp player.

Otherwise it could harm the community and the name of Winamp, which could hurt Winamp and Llama Group.

That's it. 

People saying that they want to benefit from free work for commercial purposes are totally wrong. They are just giving the chance to Winamp player to continue its journey with help of the community. 

Press release:

September 24, 2024 02:30 AM Eastern Daylight Time
BRUSSELS--([BUSINESS WIRE](https://www.businesswire.com/))--Regulatory News:

Llama Group SA (Paris: $ALLAM) (Brussels: $ALLAM)

As previously announced, Winamp will open the code of its historic Windows player on September 24 at 04:00 PM to enable the entire developer community to participate in maintaining and developing new features. Tens of millions of users still use Winamp.

Winamp plans to release two major official versions per year with new features to guarantee secure use of the software.

The player will, of course, remain free.

As far as Winamp for Creator is concerned, September 24 marks the end of the first phase in terms of products, with the delivery of licensing. Today, Winamp for Creator enables an artist or label to manage digital distribution, copyright management, licensing, NFT creation, personal subscriptions, and monetization on YouTube while building up a team of collaborators. The product phase begins with the development of marketing tools such as centralized management of social networks, website creation, and mobile applications.

NEXT MEETING
Half-year results - October 31, 2024

About Llama Group
Llama Group is a pioneer and leader in the digital music industry. With extensive expertise across various sectors, the group owns the iconic Winamp platform, the Bridger copyright management company, and the Jamendo music licensing company.
Llama Group's ambition is to build the future of the music industry through sustained investment in a range of innovative solutions and in the talent and skills of people who love music.
The group stands by its brand values: empowerment, access, simplicity, and fairness.
Winamp's vision is a world where a cutting-edge music platform connects artists and their fans like never before.
Bridger's mission is to support songwriters and composers by providing a simple and innovative solution for collecting royalties.
Jamendo enables independent artists to generate additional income through commercial licenses.
Finally, Hotmix offers a bouquet of more than sixty thematic and free digital radio stations.

About Winamp
With its ease of use, customization and irreverent attitude, Winamp changed forever the way the world listened to music in the 90s. This iconic player continues to be massively popular with a vibrant community of millions of users worldwide.
In 2023, Winamp launches its new platform. Incorporating its legendary player, Winamp has been remastered into the ultimate music machine that empowers not only listeners, but also creators and fans. Liberating music through the passion of its biggest fans, Winamp's mission is to build a platform that unlocks the relationship between artists and fans worldwide. To find out more, visit [www.winamp.com]

**Comment by Lili1228** [comment]
>it could harm the community and the name of Winamp, which could hurt Winamp and Llama Group.

You're on https://github.com/WinampDesktop/winamp, the most laughed at repository of this year, owned by people who are into NFTs. Can this name really be more harmed? xD

**Comment by tadghh** [comment]
"Yet, they need to keep control on what it becomes because Winamp has other projects ongoing and they must care about the security of users and quality of Winamp player."

How would unofficial forks affect the quality of Winamp itself? Im sure most users and devs would urge caution around possible security issues when providing their own versions. I guess some possible examples would be a malicious installer, but that would be caught and reported on first infection. Are there undisclosed malicious audio files, is hotdog-lmpbsk.mp3 going to infect my pc?

**Comment by TomJansen** [comment]
If your are afraid of people using the name Winamp or llama group then why not license it under the 3-clause BSD License

**Comment by ErdbeerbaerLP** [comment]
![image](https://github.com/user-attachments/assets/a47b3812-d25a-48f2-a9d7-61c84223b004)

**Comment by xanhast** [comment]
> People saying that they want to benefit from free work for commercial purposes are totally wrong. They are just giving the chance to Winamp player to continue its journey with help of the community. 

that isn't at all what people are arguing for - you are completely strawmanning the situation with such an effort that reads like pure shillism. and yes, the main kickback is due to the potential for this project to become for profit based on taking legal action against teenagers, something that winamp would know a lot about working in the digital music industry during the 90s, maybe they feeling nostalgic.

The license was literal entrapment and still is.  technically anyone providing a patch to this project is still breaking the license.  anyone saying "chill, it doesn't matter" should be considered a bot lubricating the honeypot.

**Comment by YarosMallorca** [comment]
>People saying that they want to benefit from free work for commercial purposes are totally wrong. 

Can you please **STOP** speaking like that of _other people_?
If Winamp said "we're going opensource" and than published this crap with a restrictive license, THIS is not Open Source, this is [Openwashing](https://en.wikipedia.org/wiki/Openwashing).

Many companies have many open source products and projects that can be used for commercial purposes along closed source projects, such as Google. It does NOT harm their rep in any way.

> They are just giving the chance to Winamp player to continue its journey with help of the community.

I would agree with them, but they're not doing that. They prohibited forking(?) which seems to be now removed, not sure.
This goes against GitHub's TOS, discussed in #6 and #16

**Comment by NoPlagiarism** [comment]
> They prohibited forking(?) which seems to be now removed

It's still forbidden

> - **No Distribution of Modified Versions**: You may not distribute modified versions of the software, whether in source or binary form.
> - Official Distribution: **Only the maintainers** of the official repository **are allowed to distribute the software and its modifications**.

Forking and changing the code (even to send PR) means violating this License

**Comment by YarosMallorca** [comment]
This checks out, @Devilsdev1985 GitHub account 2 hours ago, specifically for this post.
I'm not shaming you for not using GitHub, but posting something like this shows that you have not been a part of the Open Source community and know nothing about the topic.

Also, since when GitHub became a StackOverflow-like community
<img src="https://github.com/user-attachments/assets/36c6d2bb-37f9-485b-b5a9-494047147868" height="300px"/>

Just look at how the tables are turning, or shall I say **llamas** 😏
<img src="https://github.com/user-attachments/assets/7a9d12a2-bc01-45ca-844a-b700be52bf9d" height="200px" />

**Comment by ErdbeerbaerLP** [comment]
![image](https://github.com/user-attachments/assets/d3d24b61-ce5b-48c5-8587-03b654282735)

**Comment by NoPlagiarism** [comment]
> to allow the community to further develop the historical player because they won't

You did not. See #6

**Comment by ZaDarkSide** [comment]
> > They prohibited forking(?) which seems to be now removed
> 
> It's still forbidden
> 
> > * **No Distribution of Modified Versions**: You may not distribute modified versions of the software, whether in source or binary form.
> > * Official Distribution: **Only the maintainers** of the official repository **are allowed to distribute the software and its modifications**.
> 
> Forking and changing the code (even to send PR) means violating this License

Not really, yes making a public fork means distribution, but for now create a private fork and don't let anyone view your private fork so you can obey the license, that means don't use the fork button in the Github website UI because that will create a public fork and that will mean also distribution because it's public and can be accessed by anyone. Easy fix if you ask me.

**Comment by LokiFawkes** [comment]
> > > They prohibited forking(?) which seems to be now removed
> > 
> > 
> > It's still forbidden
> > > * **No Distribution of Modified Versions**: You may not distribute modified versions of the software, whether in source or binary form.
> > > * Official Distribution: **Only the maintainers** of the official repository **are allowed to distribute the software and its modifications**.
> > 
> > 
> > Forking and changing the code (even to send PR) means violating this License
> 
> Not really, yes making a public fork means distribution, but for now create a private fork and don't let anyone view your private fork so you can obey the license, that means don't use the fork button in the Github website UI because that will create a public fork and that will mean also distribution because it's public and can be accessed by anyone. Easy fix if you ask me.

Github TOS D5. A public repo must allow forking and redistribution using the fork button.
This repo should be privated, deleted, or changed to a different license.

**Comment by anthonyaxenov** [comment]
Well, let's forget about license for a minute and take a look at the date of [their own press release](https://about.winamp.com/press/article/winamp-open-source-code) -- **May 16, 2024**

Since then they didn't even removed these `C O N F I D E N T I A L` signs in their code.

https://github.com/search?q=repo%3AWinampDesktop%2Fwinamp+"C+O+N+F+I+D+E+N+T+I+A+L"&type=code

What _collaborative development_ are they talking about? Are we even supposed to see this code here?

As for me this is pure slovenliness. They had about 4+ months to prepare sources but it took tree days to screw up.

**Comment by bryce-carson** [comment]
> > People saying that they want to benefit from free work for commercial purposes are totally wrong. They are just giving the chance to Winamp player to continue its journey with help of the community.
> 
> that isn't at all what people are arguing for - you are completely strawmanning the situation with such an effort that reads like pure shillism. and yes, the main kickback is due to the potential for this project to become for profit based on taking legal action against teenagers, something that winamp would know a lot about working in the digital music industry during the 90s, maybe they feeling nostalgic.
> 
> The license was literal entrapment* and still is. technically anyone providing a patch to this project is still breaking the license. anyone saying "chill, it doesn't matter" should be considered a bot lubricating the honeypot.
> 
> *by saying the licence is copyleft AND hosting it on a copyleft-centric platform then proceeding to define a licence that isn't copyleft at all.
> 
> https://github.com/WinampDesktop/winamp/blob/47633753662e3f6b326084ef5928a2c0ba4347aa/LICENSE.md?plain=1#L7

Indeed. You're _not permitted to modify_ the sources. You cannot fork the repository because that is distribution of the sources. Anyone can download from my fork; or is it GitHub distributing it? Who are they going to bring a lawsuit to?

It's logically inconsistent. If I comply with their license I cannot fork it, so I cannot create a pull request except in the worst way: manually uploading modified sources. Gross. No thanks.

It really is token "open source" of the worst kind. Free devs. NFTs. How these people get money to do this illogical crap will never cease to amaze me.

**Comment by ErdbeerbaerLP** [comment]
![image](https://github.com/user-attachments/assets/4bf4123a-732c-4e1a-bbe6-06485fb7dabc)

> MIT License
> 
> Copyright (c) 2024 ErdbeerbaerLP
> 
> Permission is hereby granted, free of charge, to any person reading this comment, to deal with the comment without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the comment, and to permit persons to whom the comment is furnished to do so, subject to the following conditions:
> 
> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the comment.
> 
> THE COMMENT IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE COMMENT OR THE USE OR OTHER DEALINGS IN THE COMMENT.

**Comment by ZaDarkSide** [comment]
> > > People saying that they want to benefit from free work for commercial purposes are totally wrong. They are just giving the chance to Winamp player to continue its journey with help of the community.
> > 
> > 
> > that isn't at all what people are arguing for - you are completely strawmanning the situation with such an effort that reads like pure shillism. and yes, the main kickback is due to the potential for this project to become for profit based on taking legal action against teenagers, something that winamp would know a lot about working in the digital music industry during the 90s, maybe they feeling nostalgic.
> > The license was literal entrapment* and still is. technically anyone providing a patch to this project is still breaking the license. anyone saying "chill, it doesn't matter" should be considered a bot lubricating the honeypot.
> > *by saying the licence is copyleft AND hosting it on a copyleft-centric platform then proceeding to define a licence that isn't copyleft at all.
> > https://github.com/WinampDesktop/winamp/blob/47633753662e3f6b326084ef5928a2c0ba4347aa/LICENSE.md?plain=1#L7
> 
> Indeed. You're _not permitted to modify_ the sources. You cannot fork the repository because that is distribution of the sources. Anyone can download from my fork; or is it GitHub distributing it? Who are they going to bring a lawsuit to?
> 
> It's logically inconsistent. If I comply with their license I cannot fork it, so I cannot create a pull request except in the worst way: manually uploading modified sources. Gross. No thanks.
> 
> It really is token "open source" of the worst kind. Free devs. NFTs. How these people get money to do this illogical crap will never cease to amaze me.

You can do private forks and work on them, you are not allowed to do public forks because that means also distributing the cod from your branch, the "no forks allowed" clause was removed from the license so that's an improvement at least. Baby steps.

**Comment by Devilsdev1985** [comment]
Obviously they made mistakes. Ok then, why being so aggressive? Rather bring your support since some of you seem to know what to do better than people in charge.

**Comment by anthonyaxenov** [comment]
> Obviously they made mistakes. Ok then, why being so aggressive? Rather bring your support since some of you seem to know what to do better than people in charge.

How can we if they forbid it explicitly?
